src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -285,280 +312,100 @@
-
-    /**
-     * Represents the {@link Key} for the {@link Direction#WEST}
-     * pores side of a {@link BlockTypes#BROWN_MUSHROOM_BLOCK} or
-     * {@link BlockTypes#RED_MUSHROOM_BLOCK} {@link BlockState}.
-     */
-    public static final Supplier<Key<Value<Boolean>>> BIG_MUSHROOM_PORES_WEST = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BIG_MUSHROOM_PORES_WEST");
+    public static final Supplier<Key<Value<Double>>> BIOME_TEMPERATURE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BIOME_TEMPERATURE");
 
     /**
-     * Represents the {@link Key} for the temperature of a biome at a block location.
+     * The blast resistance of a block.
      */
-    public static final Supplier<Key<Value<Double>>> BIOME_TEMPERATURE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BIOME_TEMPERATURE");
+    public static final Supplier<Key<Value<Double>>> BLAST_RESISTANCE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLAST_RESISTANCE");
 
     /**
-     * Represents the {@link Key} for the blast resistance of a block.
+     * The blocklight value at a {@link Location}.
+     * For the skylight see {@link #SKY_LIGHT}.
      */
-    public static final Supplier<Key<Value<Double>>> BLAST_RESISTANCE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLAST_RESISTANCE");
+    public static final Supplier<Key<Value<Integer>>> BLOCK_LIGHT = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLOCK_LIGHT");
 
     /**
-     * Represents the {@link Key} for the amount of light that is emitted by the
-     * surrounding blocks at a block location. The value scales normally from 0 to 1.
+     * TODO {@link #BLOCK_LIGHT}?
+     * The amount of light that is emitted by the surrounding blocks at a block location.
+     * The value scales normally from 0 to 1.
      *
      * <p>In vanilla minecraft is this value in steps of 1/15 from 0 to 1.</p>
      */
     public static final Supplier<Key<Value<Double>>> BLOCK_LUMINANCE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLOCK_LUMINANCE");
 
     /**
-     * Represents the {@link Key} for the {@link BlockState} of a {@link BlockOccupiedMinecart} or {@link FallingBlock}.
-     */
-    public static final Supplier<Key<Value<BlockState>>> BLOCK_STATE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLOCK_STATE");
-
-    /**
-     * Represents the {@link Key} for if a {@link Boat} can move on land.
+     * The represented block's offset of a {@link MinecartEntity}.
      */
-    public static final Supplier<Key<Value<Boolean>>> BOAT_CAN_MOVE_ON_LAND = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BOAT_CAN_MOVE_ON_LAND");
+    public static final Supplier<Key<Value<Integer>>> BLOCK_OFFSET = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "BLOCK_OFFSET");
```

Comment:
Rename this back to `MINECART_BLOCK_OFFSET`, I named it like this to prevent conflicts with possible new keys that don't use an Int.